crates/server/src/actions.rs

```diff
@@ -140,6 +149,24 @@ impl CreateActionResult {
     }
 }
 
+#[derive(Deserialize, Debug)]
+pub struct RelabelConfig {
+    pub graph_name: String,
+}
+
+impl TryFrom<Action> for RelabelConfig {
+    type Error = Status;
+
+    fn try_from(action: Action) -> Result<Self, Self::Error> {
+        serde_json::from_slice::<Self>(action.body.as_slice()).map_err(from_json_error)
+    }
+}
+
+#[derive(Serialize, Debug)]
+pub struct RelabelActionResult {
+    pub relabel_millis: u128,
```

Comment:
Gotta make room in case it takes longer than 292271023 years 😅

Reply:
who knows :) I kind of followed the example of `Duration::as_millis`